sharedbuffers/pool.py

```diff
@@ -134,6 +180,27 @@ def _pack_into(self, schema, obj, section, min_pack_buffer_cell=None, idmap=None
 
     def pack(self, schema, obj, min_pack_buffer=DEFAULT_PACK_BUFFER, clear_idmaps_on_new_section=True,
             pack_buffer=None):
+        """
+        Add an object to the pool, and return a proxy to it.
+
+        :param Schema schema: The :class:`~sharedbuffers.mapped_struct.Schema` of the object data being pushed
+
+        :param obj: Object to be packed into the pool.
+
+        :param int min_pack_buffer: *(optional)* Minimum required free space in the section. If the section contains
+            less than this amount of free space, a new empty section is allocated without even trying
+            to pack the object in the almost-full section.
+
+        :param bool clear_idmaps_on_new_section: *(default True)* Whether to clear the :term:`idmap` s of full sections
+            when new sections are allocated. Doing this can keep memory usage low, but prevent efficient reuse of free
+            section space. The default is usually ok.
```

Comment:
s/ok/True/g?

Also, why `usually`?

Reply:
"The default is usually True"? It's not what I'm trying to say.